### PR TITLE
Update the install script to create /var/lib/teleport/plugins/ 

### PR DESCRIPTION
**Pull request by xnutsive**

### CODE REVIEW

File: access/slackbot/install [hunk 1]
```diff
@@ -5,10 +5,18 @@
...
+# and certificates
+#
+DATADIR=/var/lib/teleport/plugins/slackbot
+
 [ ! $(id -u) != "0" ] || { echo "ERROR: You must be root"; exit 1; }
 cd $(dirname $0)
 mkdir -p $BINDIR
 cp -f slackbot $BINDIR/ || exit 1
 
+mkdir -p $DATADIR
```
Comment (marshall-lee) [inline access/slackbot/install]: You forgot `|| exit 1` part :)

File: access/slackbot/install [hunk 2]
```diff
@@ -5,10 +5,18 @@
...
 
+#
+# the directory where Teleport Plugins store their data files
+# and certificates
+#
+DATADIR=/var/lib/teleport/plugins/slackbot
+
 [ ! $(id -u) != "0" ] || { echo "ERROR: You must be root"; exit 1; }
 cd $(dirname $0)
 mkdir -p $BINDIR
```
Comment (marshall-lee) [inline access/slackbot/install]: Here need to check the result too

File: access/slackbot/install (hunk 1 above)
Comment (marshall-lee) [reply access/slackbot/install]: Also it's better to place this command *after* the `Teleport Slackbot binaries have been copied to $BINDIR` message.

Comment (marshall-lee) [reply access/slackbot/install]: Either you could do `mkdir $BINDIR $DATADIR` at once. It case of failure, `mkdir` prints what particular write operation haven't succeed

Comment (xnutsive) [reply access/slackbot/install]: Thanks for catching this, it'd make the script more elegant.

**Comment by xnutsive** [comment]
@marshall-lee thank you for reviewing it and pointing out possible issues! Fixed both.

**Pull request merged**